rpm/tarantool.spec

```diff
@@ -167,7 +167,7 @@ C and Lua/C modules.
          -DSYSTEMD_UNIT_DIR:PATH=%{_unitdir} \
          -DSYSTEMD_TMPFILES_DIR:PATH=%{_tmpfilesdir} \
 %endif
-%if 0%{?fedora} >= 33
+%if (0%{?fedora} >= 33 || 0%{?almalinux} >= 8)
```

Comment:
It should be enabled since 9 version only. Also, please put this commit as the first one in the series.

Reply:
Fixed :rocket: